sql-jdbc/src/TableauConnector/opensearch_sql_jdbc/dialect.tdd

```diff
@@ -63,16 +63,87 @@
             <argument type='datetime' />
             <argument type='datetime' />
         </function>
+
+        <function group='operator' name='==' return-type='bool'>
+            <formula>(TIMESTAMP(%1) = TIMESTAMP(%2))</formula>
```

Comment:
Do you need to have the timestamp function when the type itself is already a datetime?

Reply:
Unfortunately, yes. Due to https://github.com/opensearch-project/sql/issues/259 and https://github.com/opensearch-project/sql/issues/294 OSSQL doesn't return valid types and can't compare across different types. Once everything is compared to TS, we can compare things.

Reply:
I think it would be better to use CAST vs. the TIMESTAMP function as mentioned in the past due to perf concerns.

Reply:
See fix in 96ac3ca0